templates/user/my_donation.html.twig

```diff
@@ -15,18 +15,20 @@
 
     {% if donations %}
         <div class="b__nudge--top b__nudge--bottom">
-            Merci pour votre engagement à nos côtés ! <br/>
+            Merci pour votre engagement à nos côtés ! <br/><br/>
             Votre dernier don a été fait {{ donations|first.donatedAt|ago }}.
             {% if subscribed_donations %}
                 <a href="{{ path('donation_subscription_cancel') }}">
                     {{ 'adherent.stop_donation'|transchoice(subscribed_donations|length) }}
                 </a>. <br />
                 {% for subscribed_donation in subscribed_donations %}
-                    Prochaine échéance du don mensuel de {{ subscribed_donation.amountInEuros }}€ le {{ subscribed_donation.nextDonationAt|date('d M Y') }}
+                    Prochaine échéance du don mensuel de {{ subscribed_donation.amountInEuros }}€ le {{ subscribed_donation.nextDonationAt|localizeddate('medium', 'none') }}
                     {% if not loop.last %}
                         <br />
                     {% endif %}
                 {% endfor %}
+            {% elseif last_subscription_ended %}
+                Vous avez mis fin à votre don mensuel le {{ last_cancel_subscription.subscriptionEndedAt|localizeddate('medium', 'none') }}. Vous pouvez en refaire un <a href="{{ path('donation_index') }}">en cliquant ici</a>.
```

Comment:
This variable does not exists. It seems that after changing this line https://github.com/EnMarche/en-marche.fr/pull/2409/files#diff-7afab20efac876cf49719e435cc8f688R49 you forget to change variable name.

Reply:
My brain is totaly disconnected from earth this morning ^^

Reply:
Stop finding yourself excuses OMG